### ISSUE TITLE
Allow AIG -> DFA with more than one input/output

### ISSUE THREAD

**Issue by wonhyukchoi** [mvcisback/py-aiger-dfa#1]
Currently, `aig2dfa` has the following restriction:
```
    The AIGBV circuit must have exactly 1 input, output, and latch
    which have names given by action_str, output_str, and state_str
    resp.
```

Is it possible to allow the AIGER circuit to have multiple inputs and outputs?

**Comment by mvcisback** [comment]
@wonhyukchoi Yes, it should be pretty straightforward, but I'm bandwidth limited at the moment so it might take a few days.

Is this time sensitive?

**Comment by wonhyukchoi** [comment]
No, not time sensitive.

Thank you for the tool and library!

**Comment by mvcisback** [comment]
Sorry this took so long. Will be looking into this in the next week.

**Comment by mvcisback** [comment]
Merged in 0.3.
